modules/@apostrophecms/express/index.js

```diff
@@ -554,12 +554,13 @@ module.exports = {
           }
           if (!sessionOptions.store.name) {
             // require from this module's dependencies
-            Store = require('connect-mongo')(expressSession);
+            const MongoStore = require('connect-mongo');
+            sessionOptions.store = MongoStore.create(sessionOptions.store.options);
```

Comment:
Looks like this line is now unnecessarily duplicated and the original version of it would be OK again.

Reply:
Actually no because connect-mongo contains a breaking change.

> Create the store using `MongoStore.create(options)` instead of `new MongoStore(options)`

Previously `MongoStore` the class expected a reference to the express session object, now it is no longer a class and we must call the `create` method and pass the store options instead.